### DISCUSSION TITLE
Socket connected but no image.

### DISCUSSION THREAD

**Discussion by zyxzxy** [bitcraze/discussions#1650] [Q&A]
hi，
I try to run Flash Wifi Example. But it runs very laggy, and after a few frames of images are transmitted, no new images are sent out. After many attempts there isn't even an image anymore. It's normal for me to use another drone. Is there something wrong with the hardware? One more question, why is the download available to my client blank?
![Screenshot from 2024-11-13 14-41-41](https://github.com/user-attachments/assets/1a0b1925-ee90-4f3d-984c-94beaabd0753)

![Screenshot from 2024-11-13 19-53-40](https://github.com/user-attachments/assets/44348fb2-1679-4789-be24-5a0ae4b26e6a)

**Comment by gemenerik** [comment]
> It's normal for me to use another drone.

Do you mean that it works OK on another drone?

> One more question, why is the download available to my client blank?

Our releases are fetched directly from the internet, specifically from the GitHub repository release: https://github.com/bitcraze/crazyflie-release.

While you're able to post this question here, the specific files for the release might be hosted on a server that’s currently unreachable from the computer/cfclient. Please ensure that the client is allowed network access to GitHub.
